arr008: Uncaught exception ghc-internal:GHC.Internal.Exception.ErrorCall:

Ix{Int}.index: Index (2) out of range ((0,1))
